src/main/java/com/arpnetworking/metrics/impl/TsdMetrics.java

```diff
@@ -342,6 +321,14 @@ private boolean assertIsOpen(final boolean isOpen) {
         return isOpen;
     }
 
+    static double convertTimeUnit(final double valueFrom, final TimeUnit unitFrom, final TimeUnit unitTo) {
+        if (unitFrom.ordinal() < unitTo.ordinal()) {
```

Comment:
Relying on the enum ordering seems pretty wonky. Could we instead
```java
double toPerFrom = unitTo.convert(1, unitFrom)
if (toPerFrom == 0) {
    toPerFrom = 1 / unitFrom.convert(1, unitTo)
}
return valueFrom * multiplicativeFactor
final double result = valueFrom * toPerFrom
```
or something semantically equivalent?

Reply:
Yeah, that's definitely better.